flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -2292,6 +2292,16 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
                     .runtimeProvided()
                     .build();
 
+    public static final BuiltInFunctionDefinition HASHCODE =
```

Comment:
sorry for the late comment but I just noticed that this field should be called `INTERNAL_HASHCODE` and be moved down to the other `INTERNAL_` functions